frontend/app/common/fetcher.ts

```diff
@@ -118,3 +118,4 @@ const createFetcher = (baseUrl: string = ''): Methods => {
 export const apiFetcher = createFetcher(`${BASE_URL}${API_BASE}`);
 export const authFetcher = createFetcher(`${BASE_URL}/auth`);
 export const adminFetcher = createFetcher(`${BASE_URL}${API_BASE}/admin`);
+export default createFetcher;
```

Comment:
I thought the whole idea was to keep `createFetcher` private? I propose to stop using it in the test and start using one of the pre-created fetchers instead.

Reply:
okay, got it